src/main/java/org/eclipse/pass/schema/SchemaMerger.java

```diff
@@ -1,22 +1,119 @@
+/*
+ * Copyright 2022 Johns Hopkins University
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
 package org.eclipse.pass.schema;
 
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.Iterator;
 import java.util.List;
+import java.util.Set;
+
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ArrayNode;
+import com.fasterxml.jackson.databind.node.ObjectNode;
 
 /**
  * The SchemaMerger class merges a list of SchemaInstance objects into a single
  * SchemaInstance object
  */
 public class SchemaMerger {
 
+    private ObjectNode merged;
+    private Set<String> ignorable = new HashSet<>(Arrays.asList("title", "description", "$id", "$schema", "$comment"));
+    private List<JsonNode> schemasToMerge;
+
+    public SchemaMerger(List<JsonNode> repository_schemas) {
+        merged = new ObjectMapper().createObjectNode();
+        schemasToMerge = repository_schemas;
+    }
+
     /**
-     * Merges a list of SchemaInstance object into a single one
+     * Merges a list of SchemaInstance object into a single one. Field:value pairs
+     * should only be added if not already present
      *
      * @param schemas list of schemas to be merged
      * @return SchemaInstance of merged schemas
      */
-    private SchemaInstance mergeSchemas(List<SchemaInstance> schemas) {
-        // TODO: Not implemented yet
-        return null;
+    JsonNode mergeSchemas() throws MergeFailException {
```

Comment:
This method should take the schemas as an argument. The schemas should not be a class field.